src/test/scala/mesosphere/marathon/health/HealthCheckTest.scala

```diff
@@ -31,6 +31,29 @@ class HealthCheckTest extends MarathonSpec {
     assert(10 == proto.getGracePeriodSeconds)
     assert(60 == proto.getIntervalSeconds)
     assert(0 == proto.getMaxConsecutiveFailures)
+    assert(!proto.hasOverridePort)
+  }
+
+  test("ToProtoOverridePort") {
```

Comment:
Can you also add the field in MergeFromProto to make sure that the overridePort is read when loading protocol buffers?